examples/reinforcement_learning_controllers/stable_baselines3_dqn_disc_pmsm_example.ipynb

```diff
@@ -411,6 +412,15 @@
     "Once you've setup the environment and defined your parameters starting the training is nothing more than a one-liner. For each algorithm all you have to do is call its ```learn()``` function. However, you should note that the execution of the training can take a long time. Currently, Stable-Baselines3 does not provide any means of saving the training reward for later visualization. Therefore, a ```RewardLogger``` callback is used for this environment (see code a few cells above)."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "motor_dashboard.initialize()"
```

Comment:
The jupyter kernel dies after successful execution of the cell.

![image](https://user-images.githubusercontent.com/95769661/151205912-2ffc2678-2c60-4e41-823b-1abdde94fa6e.png)

I tried to execute the example in PyCharm. Also no success and no error message.

Reply:
Interesting. On my machine it works fine. Not only in the browser but also in VSCode. However, I just saw that the training of the agent runs without rendering. Therefore, this cell was deleted.

Reply:
This is a problem of my environment, because some package versions are incompatible with stable basline. If I want to use stable baseline, I have to set up a new environment.